packages/cactus-plugin-ledger-connector-xdai/src/test/typescript/integration/openapi/openapi-validation.test.ts

```diff
@@ -401,19 +401,19 @@ test(testCase, async (t: Test) => {
         fake: 4,
       };
       await apiClient.runTransactionV1(
-        (parameters as any) as RunTransactionV1Request,
+        (parameters as unknown) as RunTransactionV1Request,
```

Comment:
@micoferdinand98 IIRC this cast can be eliminated entirely. Could you please double check?